app/src/components/Blog/Pagination.jsx

```diff
@@ -6,17 +6,30 @@ import ReactPaginate from 'react-paginate';
 class PaginationBlog extends Component {
 
   render() {
-    return (<section className={pagination['c-pagination-blog']}>
-      <div>
-        <ReactPaginate
-          previousLabel={<img className={pagination['left-triangle']} alt={'triangle icon'} src={boxtriangle} />}
-          nextLabel={<img className={pagination['right-triangle']} alt={'triangle icon'} src={boxtriangle} />}
-          pageNum={10}
-          activeClassName={pagination['page-selected']}
-        />
-      </div>
-    </section>);
+    if (this.props.pages === 0) {
+      return null;
+    }
+
+    return (
```

Comment:
This component doesn't render current page correctly at page load, when page > 1.